app/translations.coffee

```diff
@@ -14,12 +14,13 @@ translations =
     'searching-position': 'Paikannetaan...'
     'own-position': 'Nykyinen sijaintisi'
     'no-position': 'Ei sijaintia'
-    'origin': 'Mistä? - osoite tai pysäkki'
+    'origin-placeholder': 'Mistä? - osoite tai pysäkki'
+    'origin': 'LÄHTÖPAIKKA'
     'to-stop': 'Pysäkille'
     'line': 'Linja'
     'leaves': 'Lähtee'
     'next': 'Seuraava'
-    'destination': 'Määränpää'
+    'destination': 'MÄÄRÄNPÄÄ'
```

Comment:
These should be made uppercase using css style text-transform
- origin
- destination
- etc